tests/mbt/driver/setup.go

```diff
@@ -397,21 +397,15 @@ func (s *Driver) ConfigureNewPath(consumerChain, providerChain *ibctesting.TestC
 	// their channel, and are ready for anything to happen.
 	s.consumerKeeper(consumerChainId).SetProviderChannel(s.ctx(consumerChainId), consumerEndPoint.ChannelID)
 
-	// Commit a block on both chains, giving us two committed headers from
-	// the same time and height. This is the starting point for all our
-	// data driven testing.
+	// Commit a block on the consumer to get a header
```

Comment:
Why do we have this change here from `provider` to `consumer`?

Reply:
I removed an extra block production for the provider here because I thought I could make it so model and system heights would line up, if I managed to remove this (but it turns out there are waaay more places where blocks are produced than I thought), so the comment needed to change, too

Reply:
Turns out I was able to remove the extra block for the consumer, too, so now the comment is completely gone :)